README.md

```diff
@@ -199,6 +199,8 @@ The custom configuration options for the REST API are listed below:
   to another dictionary mapping ocp_version label to a binary image pull specification.
   This is useful in setting up customized binary image for different index image images thus
   reducing complexity for the end user. This defaults to `{}`.
+* `IIB_INDEX_TO_GITLAB_PUSH_MAP` - the mapping, `dict(<str>:<str>)`, to specify which index
+  images (keys) which should have its catalog pushed into a GitLab repository (value). 
```

Comment:
We should add a line `This defaults to `{}`.` to be consistent with other descriptions :)

Reply:
I've updated it, PTAL again